bin/make-kexec-bundle

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+CONFIG=${1:-config.nix}
+shift
+BUNDLE=$( nix-build --no-out-link '<nixpkgs/nixos>' -A config.system.build.kexec_bundle -I nixos-config=lib/kexec.nix -I nixcfg=${CONFIG} "$@" )
```

Comment:
Do we care about the current working directory? `lib/kexec.nix` is a relative path.

Reply:
We can leave this out-of-scope of this pr.

Reply:
I dislike the whole *one executable per target* setup, but I'm not going to refactor that now. Feel free to merge or ignore